ipatests/test_ipalib/test_config.py

```diff
@@ -563,7 +563,7 @@ def test_finalize_core(self):
         # Test using DEFAULT_CONFIG:
         defaults = dict(constants.DEFAULT_CONFIG)
         (o, home) = self.finalize_core(None, **defaults)
-        assert list(o) == sorted(defaults)
+        assert set(o).issuperset(set(defaults))
```

Comment:
You can just add the `fips_mode` into ipalib/constants and set its value to `object`. We had to do this workaround when @tiran added `nss_dir` to the config (see https://github.com/freeipa/freeipa/blob/master/ipalib/constants.py#L228). The test in the original from will then just work(R).